docs/chefs-identity-provider-changes.md

```diff
@@ -4,26 +4,27 @@ Within the CHEFs application a user's identity provider determines a lot of thei
 
 A User's Identity Provider (IDP) is who vouches for them. In a simplified manner: they provide a username and password (generally) and an Identity Provder verifies them and they end up with a token. Currently for CHEFs we have 3 Identity Providers: `IDIR`, `BCeID Basic` and `BCeID Business`. `IDIR` is for employees/contractors on the BC Government. In CHEFs, the `IDIR` Identity Provider allows for greater power within CHEFs; as far as the CHEFs application is concerned IDIR is the `primary` Identity Provider.
 
-Previously, all IDP logic was hardcoded within the frontend code and was difficult to change and maintain. 
+Previously, all IDP logic was hardcoded within the frontend code and was difficult to change and maintain.
```

Comment:
FYI: added a ticket to Jira so that we're more consistent with when Prettier is run.
https://bcdevex.atlassian.net/browse/FORMS-1475